src/avk.cpp

```diff
@@ -4544,12 +4575,17 @@ namespace avk
 
 		// 1. Compile the array of vertex input binding descriptions
 		{
-			// Select DISTINCT bindings (both vertex and instance inputs):
-			auto bindings = from(aConfig.mInputBindingLocations)
-				>> select([](const input_binding_to_location_mapping& bindingData) { return bindingData.mGeneralData; })
-				>> distinct() // this will invoke operator ==(const vertex_input_buffer_binding& left, const vertex_input_buffer_binding& right), we have selected vertex_input_buffer_binding in the line above
-				>> orderby([](const vertex_input_buffer_binding& generalData) { return generalData.mBinding; })
-				>> to_vector();
+			// Select bindings as vector:
+			auto bindings = aConfig.mInputBindingLocations
+				| std::views::transform([](const auto& bindingData) { return bindingData.mGeneralData; })
+				| to_vector();
+
+			// Sort bindings:
+			std::ranges::sort(bindings, std::ranges::less(), &avk::vertex_input_buffer_binding::mBinding);

```

Comment:
Wouldn't a simple `std::ranges::sort(bindings);` suffice?

Reply:
I have modeled it after the cpplinq version to only use `avk::vertex_input_buffer_binding::mBinding` for sorting. To just use `std::ranges::sort(bindings);` with the default comparison function, we need to define all comparison overloads for `avk::vertex_input_buffer_binding` as `std::ranges::less()` requires all comparison operators to be valid. Instead of defining every single one I have now used an overload of `operator <=>`.